### DISCUSSION TITLE
Github Repository Count Mismatch!

### DISCUSSION THREAD

**Discussion by rafathossain** [community/community#22515] [New to GitHub]
<p>Hello,</p>
<p>In github repository page, total repository count was showing all previously but the count showing now is the public repos only.</p>
<p>Previously Total Count = Public + Private Repo<br>
Current Total Count = Only Public Repo</p>
<p>Private repos are not being counted in all repo list! What can be the issue of this?</p>
<p>Note: I am checking the page as logged in user. I am aware that logged out user can only see the public repo count.</p>
<p>Thanks</p>

**Comment by ksgriggs** [comment]
<p>I have the same issue.  I haven’t found an answer to this as of yet.  The repository count used to include my private repos but now it is only showing public.  Like you, I am also logged in.</p>

**Comment by kategrytsenko** [comment]
<p>I have a similar issue, before yesterday all my repositories except two public were private, I’ve made all my repositories public and the total count wasn’t changed, it’s still 2. Besides that, after I pushed some changes into my new public repository - the contribution didn’t change too.</p>

**Comment by rafathossain** [comment, marked as answer]
<p>The issue has been solved by Github.</p>
